### ISSUE TITLE
Solving environment: failed  ResolvePackageNotFound:

### ISSUE THREAD

**Issue by MengHao666** [kennymckormick/pyskl#162]
when I run "conda env create -f pyskl.yaml", I meet the error as follows:

`Collecting package metadata (repodata.json): done
Solving environment: failed

ResolvePackageNotFound:
  - mkl_fft==1.3.1=py37hd3c417c_0
  - tk==8.6.10=hbc83047_0
  - libunistring==0.9.10=h27cfd23_0
  - freetype==2.11.0=h70c0345_0
  - yaml==0.2.5=h7b6447c_0
  - libstdcxx-ng==9.3.0=hd4cf53a_17
  - zstd==1.4.9=haebb681_0
  - intel-openmp==2021.4.0=h06a4308_3561
  - zlib==1.2.11=h7b6447c_3
  - libtasn1==4.16.0=h27cfd23_0
  - openssl==1.1.1t=h7f8727e_0
  - libffi==3.3=he6710b0_2
  - cryptography==35.0.0=py37h5d9358c_0
  - libgomp==9.3.0=h5101ec6_17
  - mkl_random==1.2.2=py37h51133e4_0
  - mkl-service==2.4.0=py37h7f8727e_0
  - certifi==2022.12.7=py37h06a4308_0
  - gnutls==3.6.15=he1e5248_0
  - ninja==1.10.2=h06a4308_5
  - numpy-base==1.21.5=py37hf524024_2
  - bzip2==1.0.8=h7b6447c_0
  - zeromq==4.3.4=h2531618_0
  - gmp==6.2.1=h295c915_3
  - libuv==1.40.0=h7b6447c_0
  - mkl==2021.4.0=h06a4308_640
  - lcms2==2.12=h3be6417_0
  - ninja-base==1.10.2=hd09550d_5
  - python==3.7.11=h12debd9_0
  - libtiff==4.1.0=h2733197_1
  - libgcc-ng==9.3.0=h5101ec6_17
  - cffi==1.14.6=py37hc58025e_0
  - ffmpeg==4.3=hf484d3e_0
  - jpeg==9b=h024ee3a_2
  - idna==3.4=py37h06a4308_0
  - libsodium==1.0.18=h36c2ea0_1
  - brotlipy==0.7.0=py37h27cfd23_1003
  - lame==3.100=h7b6447c_0
  - openh264==2.1.1=h4ff587b_0
  - readline==8.1=h27cfd23_0
  - typing_extensions==4.3.0=py37h06a4308_0
  - sqlite==3.36.0=hc218d9a_0
  - cudatoolkit==11.3.1=h2bc3f7f_2
  - ld_impl_linux-64==2.35.1=h7274673_9
  - lz4-c==1.9.3=h9c3ff4c_1
  - giflib==5.2.1=h7b6447c_0
  - libiconv==1.16=h7f8727e_2
  - libpng==1.6.37=hbc83047_0
  - nettle==3.7.3=hbbd107a_1
  - pytorch==1.11.0=py3.7_cuda11.3_cudnn8.2.0_0
  - ca-certificates==2023.01.10=h06a4308_0
  - xz==5.2.5=h7b6447c_0
  - ncurses==6.2=he6710b0_1
  - libidn2==2.3.2=h7f8727e_0
  - _openmp_mutex==4.5=1_gnu
  - libwebp==1.2.0=h89dd481_0`

**Comment by kennymckormick** [comment]
Which conda version are you using? If your conda version is earlier than the version listed in the repo README, please update and retry.

**Comment by MengHao666** [comment]
> just the version you recommended.
"
C:\Users\raphaelmeng>conda -V
conda 22.9.0
"

**Comment by Tianrui-Li** [comment]
I have similar issues on Windows, but it works perfectly on Ubuntu, I think it might be because the author is using Linux.

**Comment by MengHao666** [comment]
It seems the code could not handle the scence that a man is suddenly occluded by another person. The code should adapt to handle such scence even if just it give a warning.

**Comment by AakiraOtok** [comment]
Do you solve this problem ?

**Comment by BoscoShin** [comment]
i have same problem.. i use conda 23.7.2 & Window OS

**Comment by rtitton** [comment]
Same problem on Ubuntu 20.04 ARM, conda 22.11.1

**Comment by akshayneema** [comment]
Same issue of ResolvePackageNotFound in Windows with conda conda 23.7.4

**Comment by akshayneema** [comment]
Please help

**Comment by StephenDeng01** [comment]
yep,the same problem occur to me on wins 10, i think i should use ubuntu
